src/main/java/no/sikt/sws/WorkspaceHandler.java

```diff
@@ -27,14 +30,26 @@ protected String processInput(
             RequestInfo request,
             Context context
     ) throws ApiGatewayException {
+        ObjectMapper objectMapper = new ObjectMapper();
 
-        var workspace = RequestUtil.getWorkspace(request);
+        String workspace = RequestUtil.getWorkspace(request);
 
+        try {
+            JsonNode indexList = objectMapper.readTree(getIndexList(workspace));
+
+            WorkspaceResponse response = new WorkspaceResponse(workspace, indexList);
```

Comment:
Change the Output of the lambda to WorkspaceIndex isntead, and you can just return this object